### DISCUSSION TITLE
How to manage custom code?

### DISCUSSION THREAD

**Discussion by Tychodewaard** [vanjarosoftware/Vanjaro.Platform#1096] [Q&A] [answered]
The issue is: I wanted to add some Bootstrap cards to a page. As this is not a block, I put it in the custom code block. But the only way to manage the text or the image is to edit the HTML. And my customers are not really keen on that.

In the previous version there was custom blocks. These blocks somehow picked up the code and made it editable for editors without knowledge of HTML. Is there a way to achieve that again? 
I already tried saving the custom code block and add that from the tab 'custom' but that gives the same.

**Comment by meetmandeep** [comment, marked as answer]
You can always create your custom block. Add a section to the page. Add an image to it. A heading and anything else you may need. Finally save the Section as a Custom Block and name it a Card. 

RC 2 allows you to add custom classes so you could add and reuse bootstrap classes such as '.card' if needed as well. 

Finally, we'll have the template library that will be pre-populated with hundreds of blocks including cards.

**Reply by Tychodewaard** [reply]
That solves some of the usecases. 
But something like a card where I can set a link on an image, that is not possible. Because you can not create a link to an image. (#124 )

**Reply by meetmandeep** [reply]
@Tychodewaard Simply drag a Link Block and drag an image inside it. Link block is created for this exact scenario to wrap a card or multiple blocks in an a tag.

**Reply by Tychodewaard** [reply]
Another hidden gem...Thank you so much!
